Submitty/Submitty
# [Feature:System] Can Email Anyone

### What is the current behavior?
Currently, Submitty's email API only allows emails to be sent to people with a Submitty user id.

### What is the new behavior?
Now, developers can send emails to anyone. This PR also adds emailing the sysadmin when a student readds themselves to a course, see PR #9768.

### How to Test?
As an instructor, move a student to the Null section and make sure you untick "Manual." Then, as that student, go to the course's page. On the No Access page you get, click the join button. Make sure that before you dropped the student, the student had done some sort of activity in the course within the past three days. Now, as an instructor, go to the email status page and see that one of the emails is to "Sysadmin."

### Images
![image](https://github.com/Submitty/Submitty/assets/51209504/f057306b-f6dc-4b6f-847f-28790704c682)

This PR should only be merged after PR #9768 is merged.
